### ISSUE TITLE
training with train_mixed

### ISSUE THREAD

**Issue by az-ja** [princeton-vl/RAFT#61]
Hello,
thank you very much for sharing this great work.
Unfortunately I am unable to reproduce the paper's results for Sintel-test.
I trained the network using train_mixed.sh. For Sintel-test, the clean pass I get slightly worse results and for the final pass I get much worse (10% worse) than the paper, but for the training set I get very similar epe (even lower) to the paper.

I have a few questions and I would be very thankful if you give me any hint.
1. Did you try uploading Sintel test results from the network trained with train_mix.sh? 
2. train_standard.sh is the schedule that is used for the model,whose results are shown in the paper. I wanted to ask if we can also use mixed-precision with that schedule and expect very similar results for training and **test**. Because not using mixed-precision increases the training time drastically.

Except the difference of the schedule of my model to yours, my data could be not complete and for that I won't get an error.
For chairs, number of pairs is 22232, for things: 80604, and for Sintel (mixed training): 293737.

I would appreciate any hint.
Azin

**Comment by zachteed** [comment]
Hi Azin, 

I did not upload results from mixed precision training to the Sintel leaderboard. I only evaluated the mixed precision training on KITTI and Sintel train splits after training on C+T. This gave similar results to the full precision training. I would expect that --mixed_precision should not hurt performance since I did not notice a difference in validation performance, but cannot guarantee because I only submitted the 3 models from our paper using the standard training schedule (PyTorch didn't have built in mixed precision at the time).

I will look into mixed precision fine-tuning. There may be some tweaks necessary to perform as well as single precision.

Although we fix random seeds, there is some variability in training due to the non-deterministic behavior of specific operations (like grid_sampler) which was why we reported results from the median of multiple runs. I imagine there is a similar variability on the test set, but due to submission rules, I reported the results from a single and only run.

Best,

Zach
